lib/msf/core/exploit/remote/http_server.rb

```diff
@@ -381,7 +381,14 @@ def report_user_agent(address, request, client_opts={})
     report_note(
       :host => address,
       :type => 'http.request',
-      :data => "#{address}: #{request.method} #{request.resource} #{client[:os_name]} #{client[:ua_name]} #{client[:ua_ver]}",
+      :data => {
```

Comment:
Will circle back to this; but there might be some issues here sine we've lost the string interpolation we might be attempting to store the real object into the database by mistake, instead of the string representation. It might be safer to either explicitly call `.to_s` etc on these values for consistency, or to keep the string

```
      :data => { value: "#{address}: #{request.method} #{request.resource} #{client[:os_name]} #{client[:ua_name]} #{client[:ua_ver]}" }
```